### PR TITLE
Kirsten's TTL

### PR DESCRIPTION
**Pull request by kasheehy**
Added my TTL. Can you spot the lie?

### CODE REVIEW

**Comment by kasheehy** [comment]
It's either the pickle, the taco, or the pate...

File: README.md [hunk 1]
```diff
@@ -18,6 +18,6 @@ Jon:
 
 
 Kirsten:
-1. Your response here.
-2. And here...
-3. Also here!
+1. I was burned in a terrible pickle accident.
```
Comment (kasheehy) [inline README.md:21]: Aren't pickles cold?

File: README.md [hunk 2]
```diff
@@ -18,6 +18,6 @@ Jon:
...
+2. I was hit by a taco truck and given 8 tacos as compensation.
```
Comment (kasheehy) [inline README.md:22]: That doesn't seem like enough tacos...

File: README.md [hunk 3]
```diff
@@ -18,6 +18,6 @@ Jon:
...
+3. My Uncle used to give me liver paté for Christmas.
```
Comment (kasheehy) [inline README.md:23]: That sounds ~fancy~

File: README.md (hunk 2 above)
Comment (kllaskowski) [reply README.md:22]: It must have only been a flesh wound!

File: README.md (hunk 1 above)
Comment (kllaskowski) [reply README.md:21]: This seems too weird not to have happened

File: README.md (hunk 3 above)
Comment (kllaskowski) [reply README.md:23]: This is a very uncle thing to do

File: README.md (hunk 1 above)
Comment (ponchies12) [reply README.md:21]: im suspicious !

Comment (caiquelegreat) [reply README.md:21]: How do you get burned by a pickle?

File: README.md (hunk 2 above)
Comment (ponchies12) [reply README.md:22]: i wouldn mind this

**Pull request merged**